server/persistence/kube/kube.go

```diff
@@ -42,7 +41,6 @@ func (c *Client) CreateUserWorkspace(ctx context.Context, user string, workspace
 	}()
 
 	workspace.SetNamespace(c.workspacesNamespace)
-	log.Printf("cli.Create %v ", workspace)
```

Comment:
I would like to have some kind of logging here; would get some clarity from proxy on what it's thinking it's creating.

Reply:
Added in 01e7d62c3d019785ff80f98b2bc24a25a46d210c